backend/src/core/controller.py

```
@@ -1,3 +1,48 @@
+        for event_name, callback in config_events.items():
+            self.config_manager.event_bus.subscribe(modulename, event_name, callback)
+        for event_name, callback in message_events.items():
+            self.event_bus.subscribe(event_name, callback)
+
+    def configure_change(self,data:dict[str,Any]):
+        #测试用的设置更改推送接口
+        #后续可能会改成别的形式
+        for event_type, config_data in data.items():
+            try:
+                self.config_manager.event_bus.publish(event_type, config_data)
+            except Exception as e: 
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)